contribute/documentation.md

```diff
@@ -7,7 +7,6 @@ title: Documentation Contributions
 There are several ways you can help out with the improvement of Scala documentation. These include:
 
 * API Documentation in Scaladoc
-* Code examples and tutorials in activator templates.
```

Comment:
I agree that it would be good to link people to either documentation about `new` or some kind of template. @dwijnand and/or @eed3si9n, do you have any suggestions here?